Mods/defaultTemplates/Mods/HDmod RMG Templates/Content/config/defaultTemplates/extreme ii.JSON

```diff
@@ -162,9 +162,9 @@
 			{ "a" : "4", "b" : "8", "guard" : 60000 }
 		]
 	},
-	"" :
+	"EXTREME II (ban fly/DD/orb inhibition/Castle-Necro-Conflux towns, 2 player, 3-Aug-03, midnight design) xl+":

```

Comment:
Maybe we should add separate field "description"?

Reply:
Probably. But then where to display this field? It will only show up in map description after it has been generated.